_includes/header.html

```diff
@@ -29,6 +29,7 @@
               <ul class="dropdown-menu">
                 <li><a href="{{site.entregas}}">Entregas</a></li>
                 <li><a href="{{site.notas}}">Notas</a></li>
+                <li><a href="{{site.skel}}">Esqueleto</a></li>
```

Comment:
Suena un toque raro el nombre jajaja No podemos buscar uno mejor? (skel está bueno internamente, pero como alumno me quedaría o.O)

Reply:
'repositorio de descargas' no suena tan cool

Reply:
“Descargas” a secas?

Reply:
Me suena bien lo de descargas, je

Reply:
On it